pagopa/io-app
# feat: [IOCOM-548] Tracking properties on PN_UX_SUCCESS event to track cancelled PN notification

## Short description
This PR adds properties to PN_UX_SUCCESS event to better track cancelled PN notifications.

## List of changes proposed in this pull request
- notification_status: active/cancelled
- contains_multipayment: no
- count_payment: 0/1
- contains_f24: no

## How to test
Run the application with an io-dev-api-server configuration that generates PN messages that are-cancelled/are-not-cancelled and that have/do-not-have a payment. Select such messages and check that the PN_UX_SUCCESS event is triggered with proper values.
